packages/pwa/app/components/_app-config/index.jsx

```diff
@@ -64,20 +75,66 @@ const getLocale = (locals = {}) => {
  * You can also use the AppConfig to configure a state-management library such
  * as Redux, or Mobx, if you like.
  */
```

Comment:
@bendvc after chatting with you and @kevinxh , I realized that this comment for `AppConfig` in the pwa land is not as detailed as the one in the SDK. Can we somehow improve this comment? Developers would probably look at only this PWA version and not SDK.

Maybe it can be as simple as linking to the SDK like this:

```suggestion
 * You can also use the AppConfig to configure a state-management library such
 * as Redux, or Mobx, if you like.
 * @link https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/pwa-kit-react-sdk/src/ssr/universal/components/_app-config/index.js
 */
```